main.go

```diff
@@ -29,6 +32,44 @@ func main() {
 	w.Resize(fyne.NewSize(300, 500))
 	w.SetIcon(resourceIconPng)
 
+	if deskApp, ok := a.(desktop.App); ok {
+		w.SetCloseIntercept(func() {
+			w.Hide() // don't close the window if system tray used
+		})
+
+		setupSystray(deskApp, w, store)
+	}
 	go fadeSplash(splash)
 	w.ShowAndRun()
 }
+
+func setupSystray(a desktop.App, w fyne.Window, store *cityStore) {
+	a.SetSystemTrayIcon(resourceIconPng)
+	setupSystrayMenu(a, w, store)
+
+	go func() {
+		for range time.NewTicker(time.Minute).C {
+			setupSystrayMenu(a, w, store)
+		}
+	}()
+}
+
+func setupSystrayMenu(a desktop.App, w fyne.Window, store *cityStore) {
+	times := make([]*fyne.MenuItem, len(store.list)+2)
+	times[0] = fyne.NewMenuItem("Show Nomad", func() {
+		w.Show()
+	})
```

Comment:
You don't need to wrap the function call. You can just do the following:
```suggestion
	times[0] = fyne.NewMenuItem("Show Nomad", w.Show)
```

Reply:
Oh yes oops, sorry!